includes/class-sensei-course-structure.php

```diff
@@ -699,6 +701,14 @@ private function sanitize_item( array $raw_item ) {
 		if ( 'module' === $raw_item['type'] ) {
 			if ( $item['id'] ) {
 				$term = get_term( $item['id'], 'module' );
+				// Because term may get deleted in some cases during the process of changing the course teacher
+				// which takes place before saving structure.
+				if ( ! $term && $raw_item['lastTitle'] ) {
+					// During the teacher changing process of courses, modules are created or fetched using the
+					// existing title, not the one that is sent in the course structure save process.
+					$term       = $this->get_existing_module( $raw_item['lastTitle'] );
+					$item['id'] = $term;
```

Comment:
Hm, is there a case where the `$term` might be deleted? Maybe you should check it first?